### DISCUSSION TITLE
Show your projects?

### DISCUSSION THREAD

**Discussion by LarryXO** [PlummersSoftwareLLC/NightDriverStrip#192] [Ideas]
I'd love to see some images of these builds. Is there a wiki, or image repository I'm not aware of?

**Comment by davepl** [comment]
I’ve featured them in a couple of videos over the last year but not in great detail!  The GitHub is, for better or worse, the only real reference!

**Comment by stretch911** [comment]
![EEADD71B-D90A-41BA-8F86-5C81695F394A](https://user-images.githubusercontent.com/121883463/210867602-f206babd-e3a9-431a-8d18-7f1ceb128e1b.jpeg)
Work in progress. I fried my first 2 meters of 144 ws2812b 😂

I only have single output pin working at the moment...see other thread. these are piggy backed off same pin 5.
https://youtu.be/p3gqqY2q3KE

**Reply by stretch911** [reply]
https://youtu.be/VMesEMTdjhU
30 secs per effect till my arm got tired lol. effects below. power supply MUST be right and some of the effects need tweaking... :) I'm in pretty deep at this point. :)

      new PaletteEffect(OceanColors_p, 4, 0.1, 0.0, 1.0, 0.0),
        new TwinkleStarEffect(),
        new StarryNightEffect<QuietStar>("Ocean Stars", OceanColors_p, STARRYNIGHT_PROBABILITY, 1, LINEARBLEND, 2.0, 0.0, STARRYNIGHT_MUSICFACTOR),
        new MeteorEffect(1, 1, 5, .15, .25),
        new StarryNightEffect<BubblyStar>("Bubbly Stars", USAColors_p, STARRYNIGHT_PROBABILITY, 1, LINEARBLEND, 2.0, 0.0, STARRYNIGHT_MUSICFACTOR),
        new FireFanEffect(NUM_LEDS, 1, 10, 80, 2, 7, Reverse, false, false, true),
        new ColorFillEffect(CRGB::DarkCyan, 1),
        new StarryNightEffect<QuietStar>("Rainbow Twinkle Stars", RainbowColors_p, STARRYNIGHT_PROBABILITY, 1, LINEARBLEND, 2.0, 0.0, STARRYNIGHT_MUSICFACTOR),
        new ColorCycleEffect(Sequential),
        new StarryNightEffect<BubblyStar>("Little Blooming Rainbow Stars", BlueColors_p, 1.0, 2, LINEARBLEND, 2.0, 0.0, 1), // Blooming Little Rainbow Stars
        new PaletteEffect(RainbowColors_p, 4, 0.1, 0.0, 1.0, 0.0),
        new MeteorEffect(6, 1, 25, .15, .12),
        new FireFanEffect(NUM_LEDS, 1, 10, 80, 2, 7, Reverse, false, false),
        new MeteorEffect(20, 1, 25, .15, .05),
        new BouncingBallEffect(3, true, true, 1),                
        new RainbowFillEffect(6, 2),
        new ClassicFireEffect(false, false, 20),
        new MeteorEffect(12, 1, 25, .15, .08),
        new RainbowTwinkleEffect(2.0, 14),        
        new MeteorEffect(),
